lib/websocket.js

```diff
@@ -34,6 +36,29 @@ function init (env, ctx, server) {
     , alarm: levels.WARN
   };
 
+  // This is little ugly copy but I was unable to pass testa after making module from status and share with /api/v1/status
+  function status() {
+    var versionNum = 0;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["versionNum" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1887163033/issues/source?bid=2452379&fileBranchId=3084282#l41)